releases.json

```diff
@@ -1660,5 +1660,13 @@
       "1.3.3-2",
       "1.3.3-1"
     ]
+  },
+  "dragonbox": {
+    "dependency_names": [
+      "dragonbox"
+    ],
+    "versions": [
+      "1.1.2"
```

Comment:
This needs to be `1.1.2-1`. The number after the dash signifies the "wrapdb version". There may be several for any given upstream release.